cmake/FindDPCPP.cmake

```diff
@@ -57,6 +57,10 @@ if(NOT "${DPCPP_SYCL_ARCH}" STREQUAL "")
   endif()
 endif()
 
+if("${DPCPP_SYCL_TARGET}" STREQUAL "intel_gpu_pvc")
+  list(APPEND DPCPP_FLAGS "-Xspirv-translator;--spirv-ext=+SPV_INTEL_split_barrier;-DSPV_INTEL_split_barrier;")
```

Comment:
why is `-DSPV_INTEL_split_barrier` needed / what does it do?

Reply:
This was not required so I have removed it.